source/common/config/grpc_stream.h

```diff
@@ -146,6 +127,8 @@ class GrpcStream : public Grpc::TypedAsyncStreamCallbacks<ResponseProto>,
                                     POOL_GAUGE_PREFIX(scope, control_plane_prefix))};
   }
 
+  XdsGrpcContext* const owning_context_;
```

Comment:
Yeah, thinking a bit more, why does the `GrpcStream` care that this is a stateful context? Its contract is only with the callbacks, or did I miss something here?

Reply:
Yup, agreed / done.